pylint/checkers/variables.py

```diff
@@ -2461,7 +2461,7 @@ def _loopvar_name(self, node: astroid.Name) -> None:
                 else_stmt, (nodes.Return, nodes.Raise, nodes.Break, nodes.Continue)
             ):
                 return
-            # TODO: 2.16: Consider using RefactoringChecker._is_function_def_never_returning
+            # TODO: 3.0: Consider using RefactoringChecker._is_function_def_never_returning
```

Comment:
No idea what to do at all here, it seems like a major refactor. We can do it, push it to another milestone or not do it (?)